src/module/script.js

```diff
@@ -0,0 +1,57 @@
+const ul = document.getElementById('todo_list');
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
```

Comment:
- [ OPTIONAL ] The best practice is to use `const` instead of `let` to declare JS arrays. what do you think?

Reply:
I get your point champion, sure will implement that